src/generic/treectlg.cpp

```diff
@@ -3902,7 +3903,11 @@ void wxGenericTreeCtrl::OnMouse( wxMouseEvent &event )
             {
                 // only toggle the item for a single click, double click on
                 // the button doesn't do anything (it toggles the item twice)
-                if ( event.LeftDown() )
+                // Reply: wxWidgets only sends one LeftDown() event and if you
```

Comment:
Thanks, I'm going to merge this but I don't understand the format of this comment, what is this replying to?